src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,297 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String DEFAULT_LOGIN = "Ferruccio";
+    private static final String DEFAULT_PASSWORD = "df89(exZ45";
+    private static final int DEFAULT_AGE = 18;
+    private User user;
+    private RegistrationService registrationService;
+    private StorageDao storageDao;
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setLogin(DEFAULT_LOGIN);
+        user.setPassword(DEFAULT_PASSWORD);
+        user.setAge(DEFAULT_AGE);
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_ShortLoginLengthFive_NotOk() {
+        String shortLogin = "mario";
+        user.setLogin(shortLogin);
```

Comment:
```suggestion
        user.setLogin("mario");
```
You don't need this variable here. Fix the same issue in all places